packet_test.go

```diff
@@ -251,13 +245,13 @@ func areBonjourPacketsEqual(a, b bonjourPacket) (areEqual bool) {
 	// While comparing Bonjour packets, we do not want to compare packets entirely.
 	// In particular, packet.metadata may be slightly different, we do not need them to be the same.
 	// So we only compare the layers part of the packets.
-	areEqual = areEqual && reflect.DeepEqual(a.packet.Layers(), b.packet.Layers())
+	areEqual = areEqual && reflect.DeepEqual(a.packet.Layers()[2], b.packet.Layers()[2])
```

Comment:
What's the reasoning behind this change? Should we also edit the comment?

Reply:
Mmh sorry, leftover from debugging